### ISSUE TITLE
test_res.py doesn't account for nxdomain

### ISSUE THREAD

**Issue by kimbo** [jacobgb24/doh_detection#1]
I would personally remove the check on line 21 in test_res.py (https://github.com/jacobgb24/doh_detection/blob/master/doh/test_res.py#L21). If dns.message.from_wire passes, it's pretty likely they support doh, even if the response doesn't have an answer section.

**Comment by jacobgb24** [comment]
What are you looking at my code for?

But yeah that's true, it might miss some.

 It would be pretty weird though if they didn't give an answer for example.com, and generally I want the packets I'm recording to have answers.

**Comment by kimbo** [comment]
>What are you looking at my code for?

Well, it is a pubic repo...
haha :rofl:  I started following you on GH last week and that just showed up in my feed this morning. I originally thought it was something for IMAAL, and I was like "that looks interesting". But then I found out it was for your deep learning project and I was like "wow, this is way cooler than I thought".

>It would be pretty weird though if they didn't give an answer for example.com, and generally I want the packets I'm recording to have answers.

Fair enough. Though I do recall somebody we know who does weird crap with example.com (**\*cough cough\*** CENTURY LINK **\*cough cough\***) so who knows what the rest of the world is doing.
